### PR TITLE
fix: add missing translation for roles

### LINKED ISSUE #10027: User role is shown as 'Unknown' for multiple roles in record audit page.
**Bug description:**
For multiple roles (hospital clerk, community leader), role is shown as 'unknown' in record audit page.

**Steps to reproduce:**

1. Login as hospital clerk/ community leader.
2. Create and save a declaration.
3. Navigate to record audit page.
4. Observe what is shown for that user in the role column of record history.

**Actual result:**
User role is shown as 'Unknown' for multiple roles in record audit page.

**Expected result:**
All user roles should be shown correctly.

**Screenshot/Recording:**

Image

**Tested on:**
https://v19-alpha-prod.opencrvs.dev/

### PR DESCRIPTION
**Pull request by jamil314**
## Description

Fixes [this](https://github.com/opencrvs/opencrvs-core/issues/10027#issuecomment-3311832102)

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

**Review by makelicious — APPROVED** [review]
Could we have some simple test case for this? It has broken more than a few times 😢

**Review by Zangetsu101 — APPROVED** [review]

File: e2e/testcases/v2-messages/roles-in-record-audit.spec.ts
Comment (makelicious) [inline e2e/testcases/v2-messages/roles-in-record-audit.spec.ts:1]: Very nice thanks 🙏

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,15 +18,15 @@
 
 - **Upgraded MinIO** to RELEASE.2025-06-13T11-33-47Z and MinIO Client (mc) to RELEASE.2025-05-21T01-59-54Z and ensured compatibility across both amd64 and arm64 architectures.
 
-- Remove the remnants of OpenHIM from the backup & restore scripts. [#9732](https://github.com/opencrvs/opencrvs-core/issues/9732) 
+- Remove the remnants of OpenHIM from the backup & restore scripts. [#9732](https://github.com/opencrvs/opencrvs-core/issues/9732)
+- Remove the remnants of OpenHIM from the backup & restore scripts. [#9732](https://github.com/opencrvs/opencrvs-core/issues/9732)
```
Comment (Nil20) [inline CHANGELOG.md]: Duplicate lines

File: CHANGELOG.md [hunk 2]
```diff
@@ -231,15 +231,17 @@ reviewAction.reject,The label for reject button of review action,Reject
 reviewAction.title,The title for review action,Register member
 userRole.fieldAgent,Name for user role Field Agent,Field Agent
 userRole.healthcareWorker,Name for user role Healthcare Worker,Healthcare Worker
-userRole.localLeader,Name for user role Local Leader,Local Leader
+userRole.communityLeader,Name for user role Community Leader,Community Leader
+userRole.communityLeader,Name for user role Community Leader,Community Leader
```
Comment (Nil20) [inline CHANGELOG.md]: Same here

File: CHANGELOG.md [hunk 3]
```diff
@@ -231,15 +231,17 @@ reviewAction.reject,The label for reject button of review action,Reject
...
 userRole.localRegistrar,Name for user role Local Registrar,Local Registrar
 userRole.localSystemAdmin,Name for user role Local System Admin,Local System Admin
 userRole.nationalRegistrar,Name for user role National Registrar,National Registrar
 userRole.nationalSystemAdmin,Name for user role National System Admin,National System Admin
 userRole.performanceManager,Name for user role Performance Manager,Performance Manager
 userRole.policeOfficer,Name for user role Police Officer,Police Officer
 userRole.registrationAgent,Name for user role Registration Agent,Registration Agent
-userRole.socialWorker,Name for user role Social Worker,Social Worker
+userRole.hospitalClerk,Name for user role Hospital Clerk,Hospital Clerk
+userRole.hospitalClerk,Name for user role Hospital Clerk,Hospital Clerk
```
Comment (Nil20) [inline CHANGELOG.md]: Here too

**Review by Nil20 — APPROVED** [review]
